recipe/patches/0008-Disable-registry-lookup-unless-CONDA_PY_ALLOW_REG_PA.patch

```diff
@@ -1,20 +1,22 @@
-From 1620d788f86c2c010b720818357ba67917a867e3 Mon Sep 17 00:00:00 2001
+From eaf4f3898fad6fc19ae408a501b2a75156a3882a Mon Sep 17 00:00:00 2001
 From: Ray Donnelly <mingw.android@gmail.com>
 Date: Sat, 27 Oct 2018 18:48:30 +0100
-Subject: [PATCH 08/25] Disable registry lookup unless CONDA_PY_ALLOW_REG_PATHS
+Subject: [PATCH 04/20] Disable registry lookup unless CONDA_PY_ALLOW_REG_PATHS
  is not 0
 
 Co-authored-by: Isuru Fernando<isuruf@gmail.com>
 ---
- Modules/getpath.c  | 1 +
- Modules/getpath.py | 3 ++-
- 2 files changed, 3 insertions(+), 1 deletion(-)
+ Modules/getpath.c  |    1 +
+ Modules/getpath.py |    3 +-
+ PC/getpathp.c      | 1145 ++++++++++++++++++++++++++++++++++++++++++++
+ 3 files changed, 1148 insertions(+), 1 deletion(-)
+ create mode 100644 PC/getpathp.c
```

Comment:
This patch now adds an enormous c-implementation of I don't know what exactly

Reply:
At first sight, it looks like it could be an accidental recommit of [`Modules/getpath.c`](https://github.com/python/cpython/blob/v3.12.0rc1/Modules/getpath.c), but it's not.

After a bit more digging, this file was last [present](https://github.com/python/cpython/blob/v3.10.13/PC/getpathp.c) in 3.10, and was removed by https://github.com/python/cpython/commit/99fcf1505218464c489d419d4500f126b6d6dc28. Not sure why we'd need to bring it back if we didn't need it for 3.11?

Reply:
This could be an artefact of me importing the patches from Anaconda. Maybe we dropped it in conda-forge but not on the Anaconda side. If Python doesn't ship it anymore, I feel like we should try to also not ship it and see if someone reports an issue. If so, we can re-add with a comment explaining the situation.

Reply:
Seems to work fine without. :)